services/depfu/depfu.tester.js

```diff
@@ -11,12 +11,16 @@ const isDependencyStatus = Joi.string().valid(
 export const t = new ServiceTester({ id: 'depfu', title: 'Depfu' })
 
 t.create('depfu dependencies (valid)')
-  .get('/depfu/example-ruby.json')
+  .get('/github/depfu/example-ruby.json')
   .expectBadge({
     label: 'dependencies',
     message: isDependencyStatus,
   })
 
 t.create('depfu dependencies (repo not found)')
-  .get('/pyvesb/emptyrepo.json')
+  .get('/github/pyvesb/emptyrepo.json')
   .expectBadge({ label: 'dependencies', message: 'not found' })
+
```

Comment:
Could we add a test that hits a gitlab repo too?

Reply:
Sure! I just pushed some tests. For an existing repo, Depfu provides an example repo on Gitlab, too. For a nonexistent repo, I used `fdroid/nonexistant` from https://github.com/badges/shields/blob/50c0be0029a5ebfe9b387419c10495e04790d033/services/gitlab/gitlab-release.tester.js#L44